assets/styles.css

```diff
@@ -43,7 +43,6 @@ a:hover {
   transform: translateZ(200px);
 }
 .lemurInYourFace {
-  transition: transform 750ms ease, opacity 750ms ease, visibility 800ms ease;
```

Comment:
The way to do this is with the transitionend event. This could be a new backstop feature, or one can write custom Javascript to detect the transition and use the readyEvent feature of Backstop. There is already a test of readyEvent - this 'lemurInYourFace' is supposed to be the 'Simple' test case. Demonstrating CSS  transition in the "Simple" case seems wrong. 

How about we add another issue "demo CSS transition handling" that can be done in a separate PR? I didn't introduce this problem - I'm just cleaning up something that was already flaky.

Reply:
yes that works.  We can just comment the flaky test now and create an issue.

_There is actually an article on dwb  which discusses exactly your idea — using the transition event.  The onReady test script can attach a listener and just emit a console log which triggers the shot._